server/publications/collections/packages.js

```diff
@@ -69,10 +69,13 @@ function transform(doc, userId) {
 //
 //  Packages Publication
 //
-Meteor.publish("Packages", function (shopCursor) {
-  check(shopCursor, Match.Optional(Object));
+Meteor.publish("Packages", function (shopId) {
```

Comment:
Any chance of this affecting any current place "Packages" is being called as previously written (i.e with shopCursor)... now that we're changing it to shopId string?